plugins/modules/aws_s3.py

```diff
@@ -131,6 +131,15 @@
   src:
     description:
       - The source file path when performing a PUT operation.
+      - Either I(content) or I(src) must be specified for a PUT operation. Ignored otherwise.
+    type: str
+  content:
+    description:
+      - The content to PUT into an object.
+      - This parameter will be treated as a string and converted to UTF-8 before sending it to S3.
+        To send binary data, use the I(src) parameter instead.
+      - Either I(content) or I(src) must be specified for a PUT operation. Ignored otherwise.
+    version_added: "2.10"
```

Comment:
@jillr What are we supposed to be using for 'version_added' at the minute?

Reply:
@tremble the collection version the patch will land in, using semver.  We should be targeting `version_added: 1.1.0` right now.